content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -7,101 +7,81 @@ min-kubernetes-server-version: 1.22
-log onto each Window node. HostProcess containers can run as any user that is 
-available on the host or is in the domain of the host machine, allowing administrators 
-to restrict resource access through user permissions. While neither filesystem or process 
-isolation are supported, a new volume is created on the host upon starting the container 
-to give it a clean and consolidated workspace. HostProcess containers can also be built on 
-top of existing Windows base images and do not inherit the same 
-[compatibility requirements](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) 
-as Windows server containers, meaning that the version of the base images does not need 
-to match that of the host. HostProcess containers also support 
+Administrative tasks such as installation of security patches, event
+log collection, and more can be performed without requiring cluster operators to
+log onto each Window node. HostProcess containers can run as any user that is
+available on the host or is in the domain of the host machine, allowing administrators
+to restrict resource access through user permissions. While neither filesystem or process
+isolation are supported, a new volume is created on the host upon starting the container
+to give it a clean and consolidated workspace. HostProcess containers can also be built on
+top of existing Windows base images and do not inherit the same
+[compatibility requirements](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
+as Windows server containers, meaning that the version of the base images does not need
+to match that of the host. HostProcess containers also support
 [volume mounts](./create-hostprocess-pod#volume-mounts) within the container volume.
 
 ### When should I use a Windows HostProcess container?
 
-- When you need to perform tasks which require the networking namespace of the host. 
+- When you need to perform tasks which require the networking namespace of the host.
 HostProcess containers have access to the host's network interfaces and IP addresses.
 - You need access to resources on the host such as the filesystem, event logs, etc.
 - Installation of specific device drivers or Windows services.
-- Consolidation of administrative tasks and security policies. This reduces the degree of 
+- Consolidation of administrative tasks and security policies. This reduces the degree of
 privileges needed by Windows nodes.
 
 
 ## {{% heading "prerequisites" %}}
 
 {{% version-check %}}
 
-To enable HostProcess containers while in Alpha you need to pass the following feature gate flag to 
-**kubelet** and **kube-apiserver**. 
-See [Features Gates](/docs/reference/command-line-tools-reference/feature-gates/#overview) 
-documentation for more details.
+To enable HostProcess containers while you need to pass the following feature gate flag to 
```

Comment:
In v1.23 this will be enabled by default.
Perhaps we should update this to document how to `disable` the feature? (which just requires setting the feature gate to false for kubelet and kube-apiserver)

Reply:
Reworded this to explain that it is enabled by default and added instructions to disable.